Tests/kaas/plugin/plugin_kind.py

```diff
@@ -0,0 +1,50 @@
+import shutil
+import os
+import os.path
+from interface import KubernetesClusterPlugin
+from pytest_kind import KindCluster
+import logging
+logger = logging.getLogger("plugin_kind")
```

Comment:
- please sort imports (you can use isort, for instance: `pip install isort; python3 -m isort FILE`)
- please insert at least one blank line after imports
- please always use `__name__` as argument for getLogger

Reply:
I run ` python3 -m isort FILE` against `run_plugin.py` and `plugin_kind.py`.
It should now be fixed.